test/parallel/test-inspector-tracing-domain.js

```diff
@@ -28,7 +28,7 @@ function post(message, data) {
 
 function generateTrace() {
   return new Promise((resolve) => setTimeout(() => {
-    for (let i = 0; i << 1000000; i++) {
+    for (let i = 0; i < 1000000; i++) {
```

Comment:
It was but it didn't matter due to the nature of the test and how timers functioned. I can backport this change but it's needed for this since timings changed.